pkg/plugins/docker/dockerfile/target.go

```diff
@@ -115,11 +63,7 @@ func (d *Dockerfile) TargetFromSCM(source string, scm scm.Scm, dryRun bool) (cha
 	files = append(files, file)
 
 	if changed {
-		message = fmt.Sprintf("[updatecli] Instruction '%s' from Dockerfile '%s' updated to '%s'\n",
-			d.Instruction,
-			file,
-			d.Value,
-		)
+		message = fmt.Sprintf("[updatecli] Instructions changed from Dockerfile %q:\n"+strings.Join(d.messages, "\n"), d.File)
```

Comment:
Is there any reason to do `strings.Join` to insert value inside a fmt.Sprintf? 
It seems a bit redundant to me

Reply:
Oh, the one-liner makes it hard to guess, I should adapt it with an intermediate variable.

In fact, the `strings.Join` concatenate the messages from the slice of strings `messages`.

The sprintf is only there to add the first line.

Let me update this.